Src/WitsmlExplorer.Frontend/components/Modals/ServerModal.tsx

```diff
@@ -1,49 +1,111 @@
-import { Autocomplete } from "@equinor/eds-core-react";
-import { Button, TextField } from "@material-ui/core";
-import MuiThumbUpOutlinedIcon from "@material-ui/icons/ThumbUpOutlined";
-import React, { ChangeEvent, useState } from "react";
+import { Autocomplete, Checkbox, Icon, Typography, Label, Progress } from "@equinor/eds-core-react";
+import { Button, FormControlLabel, TextField } from "@material-ui/core";
```

Comment:
if possible, `eds` would be preferred over `mui` for components. We are trying to move away from them.